tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -98,6 +98,7 @@
   private String appIdString;
   private String amHostPort;
   private FileSystem fs;
+  private FileSystem historyFs;  // FileSystem used to write history-related files like config and events
   private String tonyHistoryFolder;
   private Path jobDir = null;
   private String user = null;
```

Comment:
This naming is confusing, what user is this and what for?

Reply:
In our case, we want to write history files to NN02; that's what this `historyFs` is for. I'll expand the comment.

Reply:
Sorry, I meant the `user` variable..